stores/governance/src/main/resources/db/store/h2/V0_1100_1__init.sql

```diff
@@ -163,4 +163,17 @@ CREATE TABLE committee_member
 );
 
 CREATE INDEX idx_committee_member_slot
-    ON drep_registration (slot);
\ No newline at end of file
+    ON drep_registration (slot);
+
+CREATE TABLE constitution
```

Comment:
If we are adding an epoch/activeEpoch, that could be a primary key? There will be only one possible Constitution in an epoch